### PR TITLE
CIT-103: options endpoint updates

### PR DESCRIPTION
**Pull request by zacpez**
- Added more options to endpoint
- Added missing fields to submission flow
- Few UX and UI adjustments seen in mockups and conversations

### CODE REVIEW

File: cit3.0-web/src/components/Page/OpportunityApprovePage/OpportunityApprovePage.js [hunk 1]
```diff
@@ -1,63 +1,92 @@
...
 import OpportunityApproveCallout from "../../OpportunityApproveCallout/OpportunityApproveCallout";
+import {
+  NOTIFICATION_SUCCESS,
+  NOTIFICATION_ERROR,
+} from "../../../store/constants/notification";
+import { setNotification } from "../../../store/actions/notification";
 
 const OpportunityApprovePage = ({ id }) => {
-  const history = useHistory();
+  // const history = useHistory();
```
Comment (zacpez) [inline cit3.0-web/src/components/Page/OpportunityApprovePage/OpportunityApprovePage.js:24]: To be removed
```suggestion
```

File: cit3.0-web/src/components/Page/AddOpportunity/AddOpportunity.js [hunk 2]
```diff
@@ -57,10 +57,10 @@ export default function AddOpportunity() {
   };
 
   const history = useHistory();
-  const title1 = "Add an Opportunity";
```
Comment (HypnoMama) [inline cit3.0-web/src/components/Page/AddOpportunity/AddOpportunity.js:60]: We want to keep this. No need to change

File: cit3.0-web/src/components/Page/OpportunityDeletePage/OpportunityDeletePage.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        const opp = OpportunityFactory.createStateFromResponse(response.data);
+        dispatch(setOpportunity(opp));
+      });
+    }
+  }, []);
+
+  const handleDeleteOpportunity = (e) => {
+    deleteOpportunity(opportunity).then(() => {
+      dispatch(resetOpportunity());
+      dispatch(resetOpportunity());
```
Comment (BrendanBeachBC) [inline cit3.0-web/src/components/Page/OpportunityDeletePage/OpportunityDeletePage.js:44]: Is this function supposed to be called twice here?

Comment (HypnoMama) [inline cit3.0-web/src/components/Page/OpportunityDeletePage/OpportunityDeletePage.js:44]: This line is duplicated here

**Review by BrendanBeachBC — COMMENTED** [review]
Looks pretty good, a couple of typos and other things I wanted to mention

File: cit3.0-web/src/components/Page/ReviewOpportunity/ReviewOpportunity.js [hunk 4]
```diff
@@ -64,16 +61,14 @@ const ReviewOpportunity = ({ location }) => {
...
-      <OpportunityView data={location.state} view="all" />
+      <OpportunityView view="all" />
     </div>
   );
 };
 
-ReviewOpportunity.propTypes = {
-  location: PropTypes.shape().isRequired,
-};
+ReviewOpportunity.propTypes = {};
```
Comment (HypnoMama) [inline cit3.0-web/src/components/Page/ReviewOpportunity/ReviewOpportunity.js:71]: Should we just remove these if we aren't using them?

File: cit3.0-web/src/components/Page/AddOpportunity/AddOpportunity.js (hunk 2 above)
Comment (zacpez) [reply cit3.0-web/src/components/Page/AddOpportunity/AddOpportunity.js:60]: Fixed :)

File: cit3.0-web/src/components/Page/OpportunityDeletePage/OpportunityDeletePage.js (hunk 3 above)
Comment (zacpez) [reply cit3.0-web/src/components/Page/OpportunityDeletePage/OpportunityDeletePage.js:44]: Removed for now.

File: cit-api/pipeline/migrations/0124_landusezoning_preferreddevelopment_propertystatus.py [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+        zoning.save()
+
+    PreferredDevelopment = apps.get_model("pipeline", "PreferredDevelopment")
+    preferred_development = [
+        ["Manufacturing", "", "MANU"],
+        ["Transportation and Warehousing", "", "TRAN"],
+        ["Residential", "", "RESD"],
+        ["Agriculture", "", "ARGI"]
+    ]
+    # Add new zone
```
[the remark this answers is not in the document]
Comment (zacpez) [reply cit-api/pipeline/migrations/0124_landusezoning_preferreddevelopment_propertystatus.py:30]: Fixed

File: cit-api/pipeline/migrations/0124_landusezoning_preferreddevelopment_propertystatus.py [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+        development.code = developmentInfo[2]
+        development.save()
+
+    PropertyStatus = apps.get_model("pipeline", "PropertyStatus")
+    property_status = [
+        ["Sale", "", "SALE"],
+        ["Lease", "", "LEAS"],
+        ["Both", "", "BOTH"]
+    ]
+    # Add new zone
```
[the remark this answers is not in the document]
Comment (zacpez) [reply cit-api/pipeline/migrations/0124_landusezoning_preferreddevelopment_propertystatus.py:44]: Fixed

File: cit3.0-web/src/App.js [hunk 7]
```diff
@@ -74,47 +75,47 @@ function App() {
...
+                path="/opportunity/property-details"
                 title={getTitle("Add Property Details")}
                 layout={AuthLayout}
                 component={PropertyDetails1}
               />
               <AppRoute
                 protected
                 exact
-                path="/addOpportunity/propDetails2"
+                path="/opportunity/additional-details"
```
[the remark this answers is not in the document]
Comment (zacpez) [reply cit3.0-web/src/App.js:102]: Thanks :)

File: cit3.0-web/src/App.js [hunk 8]
```diff
@@ -126,13 +127,22 @@ function App() {
...
+              />
               <AppRoute
                 protected
                 path="/manage/opportunity/*:path"
                 layout={AuthLayout}
                 component={OpportunityApprovePage}
               />
+              <AppRoute
+                protected
+                path="/opportunities/delete/*:path"
```
[the remark this answers is not in the document]
Comment (zacpez) [reply cit3.0-web/src/App.js:142]: Updated

File: cit3.0-web/src/components/NumberRangeFilter/NumberRangeFilter.scss [hunk 9]
```diff
@@ -1,3 +1,12 @@
+.unselected {
+    color: #313132;
+    border-color: #707070;
+    border-width: 2px;
+}
+
+.selected {
+}
+
```
[the remark this answers is not in the document]
Comment (zacpez) [reply cit3.0-web/src/components/NumberRangeFilter/NumberRangeFilter.scss:9]: Nope, I thought there might have been a merge issues. Will remove

File: cit3.0-web/src/components/Page/AddOpportunity/AddOpportunity.js [hunk 10]
```diff
@@ -57,10 +57,10 @@ export default function AddOpportunity() {
...
+  const title1 = "Add a Property";
   const title2 = "Confirm Property";
   const text1 =
-    "Please drop a pin on the map or enter the address of the property you want to list as an opportunity for investors.";
+    'Enter an address or "zoom in" to the map and drop a pin on the land parcel you want to list as an opportunit for investors.';
```
[the remark this answers is not in the document]
Comment (zacpez) [reply cit3.0-web/src/components/Page/AddOpportunity/AddOpportunity.js:63]: Fixed

File: cit3.0-web/src/components/Page/OpportunityApproveListPage/ApprovalFlyoutContent.js [hunk 11]
```diff
@@ -53,6 +57,32 @@ const ApprovalFlyoutContent = ({ title, onQuery, resetFiliters, search }) => {
...
+  };
+  const handleRegionalDistrictChange = (nextRDCode) => {
+    setRegionalDistrict(nextRDCode);
+    onQuery("regional_district", nextRDCode);
+  };
+
+  const handleResetFilters = () => {
+    setStatusCode("");
+    setRegionalDistrict("");
+    resetFiliters();
```
[the remark this answers is not in the document]
Comment (zacpez) [reply cit3.0-web/src/components/Page/OpportunityApproveListPage/ApprovalFlyoutContent.js:78]: Fixed

File: cit3.0-web/src/components/Page/OpportunityDeletePage/OpportunityDeletePage.js (hunk 3 above)
Comment (zacpez) [reply cit3.0-web/src/components/Page/OpportunityDeletePage/OpportunityDeletePage.js:44]: Fixed

File: cit3.0-web/src/components/Page/ReviewOpportunity/ReviewOpportunity.js (hunk 4 above)
Comment (zacpez) [reply cit3.0-web/src/components/Page/ReviewOpportunity/ReviewOpportunity.js:71]: Yeah, I suppose, as long as lint is happy

**Review by HypnoMama — APPROVED** [review]

**Pull request merged**